F21/pream9/languageconversion.py

```diff
@@ -0,0 +1,67 @@
+def LowerCaseConverter(word,lang):
+    lang=lang
+    word=word
+
+    if(lang=='turkey'or lang=='az' or lang=='tr'):
+        for letter in word:
+            if(letter=='i'or letter=="I"):
+                I=u"\u0131"
+                print(I,end='')
+            else:
+                print(letter.lower(),end='')
+
+   
+
+    if(lang=='Greek' or lang=='el'):
+        leng=len(word)-1

```

Comment:
this assignation is also not needed as we can check directly in the for loop as range(len(word)-1)

Reply:
ok, it is a good suggestion.